backpack/core/derivatives/conv_transposend.py

```diff
@@ -77,9 +77,23 @@ def _weight_jac_mat_prod(self, module, g_inp, g_out, mat):
         return self.reshape_like_output(jac_mat, module)
 
     def _weight_jac_t_mat_prod(self, module, g_inp, g_out, mat, sum_batch=True):
-        if module.groups != 1:
-            raise NotImplementedError("Groups greater than 1 are not supported yet")
-
+        """Apply transposed Jacobian of the output w.r.t.
+        weight of the torch.nn.ConvTransposeNd layer to a matrix.
+
+        Parameters:
+        -----------
+        mat: torch.Tensor
+            Matrix the transposed Jacobian will be applied to.
+            Must have shape [V, N, C_out, H_out, ...].
+        sum_batch: bool
+            Whether to sum over the batch dimension on the fly.
+
+        Returns:
+        --------
+        result: torch.Tensor
+            Jacobian-matrix product.
+            Has shape [V, N, C_w, H_w, ...] if `sum_batch == False`.
+            Has shape [V, C_w, H_w, ...] if `sum_batch == True`."""
```

Comment:
Can you use the Google docstring? I know the file you copied this from still uses `numpy`, but we agreed on Google style (see `README-dev.md`)